lib/cog/chat/adapter.ex

```diff
@@ -139,19 +139,19 @@ defmodule Cog.Chat.Adapter do
     result
   end
 
-  def send(provider, target, message) do
+  def send(provider, target, message, thread_id) do
```

Comment:
Good idea. To keep our api surface area down it makes sense to always send our metadata field here whether or not it's empty. We can ignore it in the non-Slack adapters for now.